modules/account-lib/src/coin/near/utils.ts

```diff
@@ -25,9 +25,9 @@ export class Utils implements BaseUtils {
 
   /** @inheritdoc */
   isValidPublicKey(key: string): boolean {
-    const accountIDAddressRegex = '^(([a-z\\d]+[\\-_])*[a-z\\d]+\\.)*([a-z\\d]+[\\-_])*[a-z\\d]+$';
     try {
-      new RegExp(accountIDAddressRegex).test(key) || new KeyPair({ pub: key });
+      // this regex is from near doc, https://docs.near.org/docs/concepts/account
+      /^(([a-z\d]+[\-_])*[a-z\d]+\.)*([a-z\d]+[\-_])*[a-z\d]+$/.test(key) || new KeyPair({ pub: key });
```

Comment:
is there a length restriction of address?

Reply:
![image](https://user-images.githubusercontent.com/85512331/155382231-297a048e-f9ff-402e-abf3-082963f43569.png)
i found this in the doc, i can add this restriction in this pr

Reply:
Yes @facundoCarreSimtlix, implicit accounts have 64 characters, but account ids can have between 2 and 64. Please add that validation to the regex pattern